src/app.js

```diff
@@ -27,6 +27,7 @@ if (!environmentData.success) {
 }
 
 require('@configs')
+require('@configs/cloud-service')
```

Comment:
@ankit1shahu already config folder is added. Do we need to require cloud services separately?